src/sentry_plugins/amazon_sqs/plugin.py

```diff
@@ -29,6 +29,13 @@ def get_regions():
     return public_region_list + cn_region_list
 
 
+def track_response_metric(success):
```

Comment:
Just as a thought, since this is tracking the return of a boolean from `forward_event`, it seems this would make more sense as a decorator of the function, instead of instrumenting alongside each return statement.

I'd much rather see something like:

```python
@track_response_metric
def forward_event(...)
```

Or maybe even this can be done by the caller of `forward_event` and capture all DataForwardingPlugins? I'm not looking too deep, but I assume somewhere we have an instance of `AmazonSQSPlugin that calls `forward_event`.

So somethiing like this pseudocode:

```python
for plugin in data_forwarding_plugins:
    result = plugin.forward_event(event, payload)
    track_response_metric(plugin.slug, result)
```

Then all plugins are captured and we have the slug.

Arguably also the slug should be as a part of the metric tags.

Maybe something like:

```
metrics.incr("plugins.forward_event", instance=plugin.slug, tags={"success": success})
```